src/main/java/mat/server/service/jobs/CheckUserChangePasswordLimit.java

```diff
@@ -22,15 +21,9 @@
 import org.springframework.stereotype.Service;
 import org.springframework.ui.freemarker.FreeMarkerTemplateUtils;
 
-import freemarker.template.Configuration;
-import freemarker.template.TemplateException;
-import mat.dao.EmailAuditLogDAO;
-import mat.dao.UserDAO;
-import mat.model.EmailAuditLog;
-import mat.model.User;
-import mat.server.service.UserService;
-import mat.server.util.ServerConstants;
-import mat.shared.ConstantMessages;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.*;

```

Comment:
individual imports are preferred in MAT. could you update your IntelliJ settings?